specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json

```diff
@@ -661,8 +660,47 @@
             "description": "Accepted request for purging an Log Analytics workspace.",
             "schema": {
               "$ref": "#/definitions/WorkspacePurgeResponse"
+            },
+            "headers": {
+              "x-ms-status-location": {
```

Comment:
@alexeldeib - this looks like it's using long-running-operation mechanism but with a non-standard header name? Is this a new API or an old API that has been public for some time?

Reply:
This is an existing public API. The swagger modeling used x-ms-long-running-operation originally because I thought we were aligned with the long-running-operation mechanism, but as you've noticed that's unfortunately not the case. I can discuss the possibility of revising this with my team (maybe for a future version?) but the API is indeed live (see the .zip Fiddler files attached to the PR if you'd like an example to compare against).

Reply:
Thanks @alexeldeib! Indeed in a future version this will need to be brought in line with ARM guidelines - it would be best to consult with @ravbhatnagar.
Does that mean the other new APIs in this PR (e.g. Get status for ongoing purge operation) have also been live for some time? Or have they been added in order to support the unconventional long-running operations?

Reply:
Purge + Purge status have both existed for a while now. Originally these were modeled as two separate requests in swagger. Here's the original [Purge status operation](https://github.com/Azure/azure-rest-api-specs/blob/57d4deeb0c569c395a295f7c312aed09ad6e30d6/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json#L286).

We modified the API due to a security issue, splitting out the filter field into operator/value in #2808. Piggybacking on the breaking change I was already making, I used the chance to move us to use x-ms-long-running-operation. At the time it wasn't clear to me we didn't align to this model. 

Beyond that, the [original modeling](https://github.com/Azure/azure-rest-api-specs/blob/57d4deeb0c569c395a295f7c312aed09ad6e30d6/specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/components_API.json#L286) with the two operations split was missing the headers, which this PR adds. Apologies for making a bit of a mess here...